packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -41,7 +41,7 @@ export const suppressIconStyle = {
 };
 
 export const sendBoxStyle = mergeStyles({
-  minHeight: '0',
+  minHeight: '2.25rem', // prevents the input text box from being sized to 0px when the meeting composite chat pane is closed.
```

Comment:
Screenshot for context:
![image](https://user-images.githubusercontent.com/2684369/128934156-1f50bab7-a68d-49a5-90b3-92ae3405d608.png)